File: web/blocks/baubles/arrow.dart
// NetTango Copyright (C) Michael S. Horn, Uri Wilensky, and Corey Brady. https://github.com/NetTango/NetTango

part of NetTango;

class Arrow {

  static DivElement draw() {
    final arrow = new DivElement();
    arrow.innerText = "➔";
    arrow.classes.add("nt-arrow");
    return arrow;
  }

}

File: src/blocks/baubles/arrow.ts
// NetTango Copyright (C) Michael S. Horn, Uri Wilensky, and Corey Brady. https://github.com/NetTango/NetTango

class Arrow {

  static draw(): HTMLDivElement {
    const arrow = document.createElement("div")
    arrow.innerText = "➔"
    arrow.classList.add("nt-arrow")
    return arrow
  }

}

export { Arrow }
